### PR TITLE
Small doc improvement

### PR DESCRIPTION
**Pull request by benoittgt**
I saw missing links or invalid private class information while looking at rspec-rails documentation.

### CODE REVIEW

**Comment by benoittgt** [comment]
> +/home/travis/.rvm/gems/ruby-2.6.3/gems/bundler-2.1.3/lib/bundler/runtime.rb:312:in `check_for_activated_spec!': You have already activated bundler 2.1.2, but your Gemfile requires bundler 2.1.3. Since bundler is a default gem, you can either remove your dependency on it or try updating to a newer version of bundler that supports bundler as a default gem. (Gem::LoadError)

**Comment by benoittgt** [comment]
Bundler again. (ಠ_ಠ)

rubygems install bundler 2.1.2, then we install bundler 2.1.3 but we choose the wrong version somewhere.

I was working on a small reproduction script to try to answer to https://github.com/bundler/bundler/issues/7487#issuecomment-567248016

At the moment I do
```sh
$ rvm use 2.7 && rm -f Gemfile.lock && export RAILS_VERSION='~>6.0' && bundle --binstubs && bundle exec rake smoke:app
# no problem
$ bundle exec rake cucumber
# 💥
```
And it breaks. I will try to narrow it tomorrow. 💤

**Comment by pirj** [comment]
😢

**Comment by JonRowe** [comment]
Technically, I think its the generated sub apps that are the issue.

**Comment by benoittgt** [comment]
Ok I look a little bit and it seems there is something with [`unset_bundler_env_vars`](https://github.com/rspec/rspec-rails/blob/4b75a0d45b48ea2b747a6c420e7ebcb403b33858/features/support/env.rb#L8).

I look a little bit how _aruba_ works but I am not sure I understand how [`unset_bundler_env_vars`](https://github.com/cucumber/aruba/blob/master/lib/aruba/api/bundler.rb#L10) works.

With `unset_bundler_env_vars` commented. `bundle exec cucumber` returns [less errors](https://gist.github.com/benoittgt/d6eabfb9c9e47c4b4ebe521843aadcca). But it is not fixed yet.@

If I use the `features/support/env.rb` patch from https://github.com/rspec/rspec-rails/pull/2241 I have [even less errors](https://gist.github.com/benoittgt/b458d6a1f927e7d1df9db08e32bcc00a).

Maybe @mvz you have an idea?

**Comment by benoittgt** [comment]
I push this commit for CI testing purpose.

**Comment by benoittgt** [comment]
Ok it is working... For https://github.com/rspec/rspec-rails/pull/2250/commits/f1b5011e5d81a5a707a3a0cdf3ee21eb64b0d72f I simply copy past it. I am not confident to merge without a re-review.

**Comment by pirj** [comment]
@mvz do you mind to approve/criticise this approach?

File: features/support/env.rb [hunk 1]
```diff
@@ -4,11 +4,31 @@
 module ArubaExt
   def run_command(cmd, timeout = nil)
     exec_cmd = cmd =~ /^rspec/ ? "bin/#{cmd}" : cmd
-    # Ensure bundler env vars are unset
     unset_bundler_env_vars
-    # Ensure the correct Gemfile is found
+    # Ensure the correct Gemfile and binstubs are found
     in_current_directory do
-      super(exec_cmd, timeout)
+      with_unbundled_env do
```
Comment (mvz) [inline features/support/env.rb]: I'll chip in to say that this call to `with_unbundled_env` is probably not needed.

Comment (benoittgt) [reply features/support/env.rb]: So I tried [without](https://github.com/rspec/rspec-rails/commit/3a1f696f1241d48973f49d4c2147bf2493453f4f)

And [it breaked](https://travis-ci.org/rspec/rspec-rails/builds/632604861) the build. So I rollback.

Comment (mvz) [reply features/support/env.rb]: That's surprising and I'm a little concerned that I don't know what is going on in my own fix, but if it works for now, I guess it's ok. Thanks for checking this, anyway.

**Comment by benoittgt** [comment]
Thanks for the review you all. 

I will be happy if the CI break a little bit less. 😄

Happy new year.

**Comment by benoittgt** [comment]
I just reran the CI on master to try the last [Bundler release 2.1.4](https://github.com/bundler/bundler/releases/v2.1.4). And no issues.

**Pull request merged**